vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridHeaderFooterRowIT.java

```diff
@@ -257,6 +259,16 @@ public void addHeaderRow_setMultiselect_disableSelection() {
                 "0", headerCells.get(0).getText());
     }
 
+    @Test
+    public void multiSelectionToSingleselect() {
+        clickButton("set-singleselect");
+        grid.select(1);
+        Assert.assertTrue( "The first row of the grid should be selected", grid.getRow(1).isSelected());
+        grid.select(2);
+        Assert.assertFalse( "The first row of the grid should be not selected", grid.getRow(1).isSelected());
+        Assert.assertTrue( "The second row of the grid should be selected", grid.getRow(2).isSelected());
```

Comment:
Please remove the extra space after the `(`

Reply:
done